app/Jobs/NotifyUserOfNewGistComments.php

```diff
@@ -68,7 +68,7 @@ private function handleComment($comment, $gist, $user)
 
     private function commentNeedsNotification($comment, $user)
     {
-        if ($comment['updated_at'] < $user['created_at']) {
+        if ($comment['updated_at'] < $user['created_at'] || $comment['user']['id'] !== $user['owner']['id']) {
```

Comment:
I think you meant this to be `===`?

"Does this comment needs notification?"
"if it was updated before the user signed up" OR
"if the commenting user is the same as the gist user",
   then no.
"Otherwise, yep."